### ISSUE TITLE
Add "About" section to the UI

### ISSUE THREAD

**Issue by sgabenov** [DeviceFarmer/stf#686]
Add "About" section to the UI, where will be info:

Current Version \ Build \ Release number
Link to the GitHub Repository

**Comment by denis99999** [comment]
@sgabenov good idea, can you tell me what you mean exactly by Version, Build and Release number using an example with the last release of STF?

**Comment by sgabenov** [comment]
Hello, i mean i want to know what version of stf_build i am using, so need to put info inside app, so i could not only check from version of docker that is running, but also from UI.
E.g. - Section  "About"
Product version: v3.6.5
Build number: 123
GitHub repo: https://github.com/DeviceFarmer/stf/
